nautobot/extras/templates/extras/marketplace.html

```diff
@@ -0,0 +1,227 @@
+{% extends "base.html" %}
+{% load helpers %}
+{% load static %}
+
+{% block extra_styles %}
+<style>
+    a.tile,
+    a.tile:hover,
+    a.tile:focus,
+    a.tile:visited {
+        color: inherit;
+        text-decoration: none;
+    }
+
+    .tile-header {
+        display: block;
+    }
+
+    .app-details {
+        column-gap: 20px;
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    .app-details dt {
+        margin-top: 20px;
+    }
+
+    .app-details ul {
+        list-style: none;
+        padding: 0;
+    }
+
+    .app-details > :last-child {
+        flex: 1;
+    }
+</style>
+{% endblock %}
+
+{% block content %}
+    <h1>{% block title %}Marketplace{% endblock %}</h1>
+
+    <div class="row">
+        <div class="col-md-12">
+            <section class="tiles">
+                {% for app in apps %}
+                    {% with id=app.name|slugify %}
```

Comment:
Would using `app.package_name` as the id be preferable, once we fix the issue with some apps missing package names in the marketplace data?

Reply:
pretty much, yes. if it weren't for two missing package names for Operational Compliance and Event Bus apps, I'd happily switch this to `package_name`. for now I needed to have a unique ID for each app that's why I had to slugify name instead

Reply:
4053d831bbc2e1ae1002e5aa20dcfcaab50b7aa1